Libplanet.Tests/TestUtils.cs

```diff
@@ -300,12 +300,17 @@ public static void AssertBencodexEqual(IValue expected, IValue actual)
             );
         }
 
-        public static void AssertSorted<T>(IEnumerable<T> list)
+        public static void AssertSorted<T>(IEnumerable<T> expected)
             where T : IComparable<T>
         {
-            Assert.True(list
-                .Zip(list.Skip(1))
-                .All(pair => pair.First.CompareTo(pair.Second) <= 0));
+            List<T> list = expected.ToList();
```

Comment:
It would be better if there is a mention about why we don't use `.Zip()` here.

Reply:
Amended at https://github.com/planetarium/libplanet/pull/2532/commits/664941f54ed04cdcb87adf7144da1643c93f0c78